Simplenote/src/main/java/com/automattic/simplenote/di/AppModule.kt

```diff
@@ -0,0 +1,16 @@
+package com.automattic.simplenote.di
+
+import android.content.Context
+import com.automattic.simplenote.Simplenote
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+class AppModule {
+    @Provides
+    fun provideSimplenote(@ApplicationContext appContext: Context): Simplenote = appContext as Simplenote
```

Comment:
I've injected application context as `Simplenote` as it saves us some casting operations. Does it work for you like this or do you prefer previous behaviour?